operator/rhtap.Dockerfile

```diff
@@ -0,0 +1,52 @@
+# TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
+# We're targeting a floating tag here which should be reasonably safe to do as both RHEL major 8 and Go major.minor 1.20 should provide enough stability.
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+COPY . .
+
+RUN scripts/rhtap/fail-build-if-git-is-dirty.sh
+
+# Build the operator binary.
+# TODO(ROX-20240): enable non-release development builds.
+ENV CI=1 GOFLAGS="" GOTAGS="release" CGO_ENABLED=1
+
+RUN GOOS=linux GOARCH=$(go env GOARCH) BUILD_TAG=$(make tag) scripts/go-build.sh operator && \
+    cp bin/linux_$(go env GOARCH)/operator image/bin/operator
+
+# TODO(ROX-20312): pin image tags when there's a process that updates them automatically.
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+LABEL \
+    com.redhat.component="rhacs-operator-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Operator for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Operator for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="operator" \
+    io.openshift.tags="rhacs,operator,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-operator-rhel8" \
```

Comment:
not sure if this matters, but this differs from what's in the midstram dockerfile (`rhacs-rhel8-operator`).

Reply:
Updated it to midstream `name` label. 
There is a difference between `operator` and `roxctl` name label formatting:
- operator: `rhacs-rhel8-operator` (https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-operator/Dockerfile.in#L24) 
- roxctl: `rhacs-roxctl-rhel8` (https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-roxctl/Dockerfile.in#L26)

I think it's better to keep it like that.